src/scss/components/topnav/index.scss

```diff
@@ -62,63 +73,58 @@
       font-size: 0.813rem;
       height: 3.5rem;
       line-height: 1;
-      padding: 0 0.563rem;
       text-align: center;
       text-decoration: none;
-      text-transform: uppercase;
+      display: flex;
+      padding: 0.625rem;
     }
 
     hx-icon {
-      color: $gray-0;
+      color: $gray-600;
       font-size: 0.938rem;
-      height: 1.25rem;
+      height: 1.375rem;
       margin: 0.688rem 0 0.313rem;
       width: 1.25rem;
     }
 
     p {
-      color: $gray-0;
+      color: $gray-700;
       line-height: 1.5;
-      font-size: 0.625rem;
+      font-size: 0.875rem;
+      padding: 0.625rem;
     }
 
     a:hover {
-      background-color: $gray-900;
+      > hx-icon,
+      p {
+        color: $gray-950;
+      }
     }
 
     a.disabled,       // deprecated
     a.hxDisabled {
       @include hxTopNavControl(disabled);
-
-      &:hover {
-        @include hxTopNavControl(disabled, hover);
-      }
     }
 
     a.selected {
-      border-bottom: 0.25rem solid $red-900;
-
-      > hx-icon, p {
-        color: $gray-25;
+      > hx-icon,
+      p {
+        color: $gray-950;
       }
     }
 
     #topnav-menu:hover,                // deprecated
-    #topnavMenuId hx-menuitem:hover {  // deprecated
-      background-color: $gray-900;
-      color: $gray-0 !important;
+    #topnavMenuId hx-menuitem:hover {
+      // deprecated
```

Comment:
Please revert this change....move to Line 117.

Reply:
Changes have been done, please review.